common.blocks/button/_togglable/button_togglable.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @module button
+ */
+
+modules.define('button', function(provide, Button) {
+
+/**
+ * @exports
+ * @class button
+ * @bem
+ */
+provide(Button.decl({ modName : 'togglable' }, /** @lends button.prototype */{
+    onSetMod : {
+        'checked' : function(_, modVal) {
+            this.__base.apply(this, arguments);
+
```

Comment:
As [I see](https://github.com/bem/bem-components/blob/v2/common.blocks/button/button.js#L35) in our codebase, this empty line is redundant.